app/main.py

```diff
@@ -6,17 +6,43 @@
 
 from .i18n import active_translation
 from .knowledge_panels import KnowledgePanels
-from .models import FacetName, HungerGameFilter, Taxonomies
+from .models import FacetName, FacetResponse, HungerGameFilter, Taxonomies
 
-app = FastAPI()
+# Metadata for the API
+tags_metadata = [
+    {
+        "name": "knowledge-panel",
+        "description": "Return different knowledge panels based on the facet provided.",
+    },
+]
+description = """
+Providing knowledge panels for a particular Open Food Facts facet (category, brand, etc...)
+
+A standardized way for clients to get semi-structured but generic data that they can present to users on product pages.
+"""  # noqa: E501
+
+app = FastAPI(
+    title="Open Food Facts knowledge Panels API",
+    description=description,
+    version="0.0.1",
+    contact={
+        "name": "Slack",
+        "url": "https://openfoodfacts.slack.com/archives/C03LFRKLVBQ",
+    },
+    license_info={
+        "name": "GNU Affero General Public License v3.0",
+        "url": "https://www.gnu.org/licenses/agpl-3.0.en.html",
+    },
+    openapi_tags=tags_metadata,
+)
 
 
 @app.get("/")
 async def hello():
     return {"message": "Hello from facets-knowledge-panels! Tip: open /docs for documentation"}
 
 
-@app.get("/knowledge_panel")
+@app.get("/knowledge_panel", response_model=FacetResponse)
```

Comment:
We also need to add metadata on parameters (and you won't need to describe them in API references.md.

See https://fastapi.tiangolo.com/tutorial/query-params-str-validations/#declare-more-metadata

You have to use the Query object as a default value.

For each, you should provide a description, and eventually (when not trivial), an example.

Reply:
Perfect :)